tests/ui_tests/app/page/common/base_page.py

```diff
@@ -262,12 +272,16 @@ def wait_config_loaded(self):
         self.find_element(CommonLocators.socket, timeout=30)
         self.find_element(CommonLocators.profile, timeout=30)
 
-    def hover_element(self, locator: Locator):
+    def hover_element(self, locator: Union[Locator, WebElement]):
         """
         Moves the cursor over an element and hovers it.
         """
-        hover = ActionChains(self.driver).move_to_element(self.find_element(locator))
-        hover.perform()
+        if isinstance(locator, Locator):
+            hover = ActionChains(self.driver).move_to_element(self.find_element(locator))
```

Comment:
Here and in similar places, it'll be nice to get WebElement with if and then have the same lines of code to execute `move_to_element`. Something like:
```
element = locator if isinstance(locator, Locator) else self.find_element(locator)
hover = ActionChains(self.driver).move_to_element(element)
hover.perform()
```

Reply:
fixed